source/Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.cs

```diff
@@ -0,0 +1,67 @@
+/*
+This example shows how to acquire an HDR image from the Zivid camera with fully
+configured settings for each frame. In general, taking an HDR image is a lot
+simpler than this as the default settings work for most scenes. The purpose of
+this example is to demonstrate how to configure all the settings.
+*/
+
+using System;
+using System.Collections.Generic;
+using Duration = Zivid.NET.Duration;
+
+class Program
+{
+    static void Main()
+    {
+        try
+        {
+            var zivid = new Zivid.NET.Application();
+
+            Console.WriteLine("Connecting to the camera");
+            var camera = zivid.ConnectCamera();
+
+            Console.WriteLine("Configuring settings same for all HDR frames");
+
+            var settingsDefault = new Zivid.NET.Settings
+            {
+                Brightness = 1,
+                Bidirectional = false,
+                BlueBalance = 1.081,
+                RedBalance = 1.709
+            };
+            settingsDefault.Filters.Contrast.Enabled = true;
+            settingsDefault.Filters.Contrast.Threshold = 5;
+            settingsDefault.Filters.Gaussian.Enabled = true;
+            settingsDefault.Filters.Gaussian.Sigma = 1.5;
+            settingsDefault.Filters.Outlier.Enabled = true;
+            settingsDefault.Filters.Outlier.Threshold = 5;
+            settingsDefault.Filters.Reflection.Enabled = true;
+            settingsDefault.Filters.Saturated.Enabled = true;
+
+            Console.WriteLine("Configuring settings different for all HDR frames");
+            ulong[] iris = { 17, 27, 27 };
+            long[] exposureTime = { 10000, 10000, 40000 };
+            double[] gain = { 1.0, 1.0, 2.0 };
+            var settingsHDR = new List<Zivid.NET.Settings>();
+            for (int i = 0; i < 3; ++i)
+            {
+                settingsDefault.Iris = iris[i];
+                settingsDefault.ExposureTime = Duration.FromMicroseconds(exposureTime[i]);
+                settingsDefault.Gain = gain[i];
+                settingsHDR.Add(settingsDefault);
```

Comment:
Have you tested that this works as expected? I think here you just mutate the same Zivid.NET.Settings object and change Iris on the same object several times. I think you will end up with 3x frames that are equal. (I am trying to improve this problem with API 2).

Reply:
I had the same doubt in reviewing something similar in C++
https://github.com/zivid/cpp-extra-samples/blob/4051bb4dd466c8f696ec89c973fc06340aa661e8/source/Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.cpp#L41-L48

Reply:
done as discussed